.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -6,27 +6,27 @@ body:
     id: title-help
     attributes:
       value: |
-        > Titles should be short, descriptive, and compelling.
+        > Titles should be short, descriptive, and compelling. Use sentence case.
   - type: textarea
     id: issue-description
     attributes:
       label: Issue description and context
       description: |
-        Describe the issue so that someone who wasn't present for its discovery can understand the problem and why it matters. Use full sentences, plain language, and good [formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax). Share desired outcomes or potential next steps. Images or links to other content/context (like documents or Slack discussions) are welcome.
+        Describe the issue so that someone who wasn't present for its discovery can understand why it matters. Use full sentences, plain language, and good [formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax). Screenshots and links to documents/discussions are welcome.
```

Comment:
I think screenshots/links at the top kind of distract form the AC and bury the links, especially if the link has text associated with it. I would really prefer if we brought back the optional additional context section. I honestly missed this was on the last PR and now trying to make PRs seems more messy with this combined section and I end up just wanting to make a section anyways.

Reply:
I pulled out `additional context` into its own section. Does that address your comment?